libcudacxx/include/cuda/std/detail/libcxx/include/cstddef

```diff
@@ -82,7 +77,7 @@ _LIBCUDACXX_HIDE_FROM_ABI constexpr byte operator|(byte __lhs, byte __rhs) noexc
     static_cast<unsigned char>(static_cast<unsigned int>(__lhs) | static_cast<unsigned int>(__rhs)));
 }
 
-_LIBCUDACXX_HIDE_FROM_ABI constexpr byte& operator|=(byte& __lhs, byte __rhs) noexcept
+_LIBCUDACXX_HIDE_FROM_ABI _CCCL_CONSTEXPR_CXX14 byte& operator|=(byte& __lhs, byte __rhs) noexcept
```

Comment:
I believe those can all be constexpr?

Reply:
They cannot as they have side effects (modifying `__lhs`) which is not allowed in C++11

Reply:
Oh yeah, I hate that